src/main/scala/is/hail/variant/MatrixTable.scala

```diff
@@ -1330,9 +1330,9 @@ class MatrixTable(val hc: HailContext, val ast: MatrixIR) {
     assert(entryAST.`type`.isInstanceOf[TStruct])
 
     entryAST.toIR() match {
-      case Some(ir) =>
+      case Some(ir) if entryAST.`type`.asInstanceOf[TStruct].size < 500 =>
```

Comment:
in all these cases we have an assert isInstanceOf on an earlier line. Let's refactor the code so we do
```scala
val entryType = entryAST.`type`.asInstanceOf[TStruct]
```
and then use it here
```scala
  case Some(ir) if entryType.size < 500
```